src/cr/cube/matrix.py

```diff
@@ -1397,6 +1397,12 @@ def ordering(self):
     def values(self):
         return self._base_vector.values[self._opposing_order]
 
+    @lazyproperty
+    def margin(self):
+        if not isinstance(self._base_vector.margin, np.ndarray):
+            return self._base_vector.margin
+        return self._base_vector.margin[self._opposing_order]
```

Comment:
Please respect the existing ordering and position this property alphabetically.

Reply:
Add comments to describe why this conditional behavior is necessary.

Reply:
done